### PR TITLE
Remove the "ideas" section.

### PR DESCRIPTION
**Pull request by couet**
Remove the whole "ideas" mechanism as requested [here](https://github.com/root-project/web/issues/895)

### CODE REVIEW

File: contribute/index.md [hunk 1]
```diff
@@ -41,36 +38,12 @@ templates, or directly from your terminal:
...
-<ul>
-{% for idea in sorted %}
-{% if idea.state != "completed" %}
-<li> {{idea.date | date_to_string }} <a href="{{ idea.url | relative_url }}"> {{ idea.title}} </a><br>
-{{idea.summary}}
-</li>
-{% endif %}
-{% endfor %}
-</ul>
+### Fix an “easy” bug ?
```
Comment (Axel-Naumann) [inline contribute/index.md:43]:
```suggestion
### What about fixing an "easy" bug?
```

File: contribute/index.md [hunk 2]
```diff
@@ -41,36 +38,12 @@ templates, or directly from your terminal:
...
 
-### Completed ideas
+Several of ROOT’s bugs are actually not all that difficult to fix - we just don’t get around
+to actually do it. Maybe you can help? It’s a great way to exercise how to collaborate in a
+vivid, medium-size project, following industry best practice. Find your favorite
+[here]( https://github.com/root-project/root/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22).
```
Comment (Axel-Naumann) [inline contribute/index.md:48]:
```suggestion
[here](https://github.com/root-project/root/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22).
```

**Review by Axel-Naumann — APPROVED** [review]
LGTM, but see comments.

File: contribute/index.md [hunk 3]
```diff
@@ -41,36 +38,12 @@ templates, or directly from your terminal:
...
-<li> {{idea.date | date_to_string }} <a href="{{ idea.url | relative_url }}"> {{ idea.title}} </a><br>
-{{idea.summary}}
-</li>
-{% endif %}
-{% endfor %}
-</ul>
+### What about fixing an "easy" bug?
 
-### Completed ideas
+Several of ROOT’s bugs are actually not all that difficult to fix - we just don’t get around
```
Comment (eguiraud) [inline contribute/index.md:45]: ...if you are a good C++ programmer that knows their way around a debugger and can handle large code bases :)

Comment (couet) [reply contribute/index.md:45]: Do we really want to add this sentence? it sounds a bit "elitist".  What about something more neutral:

_Several of ROOT’s bugs are actually not all that difficult to fix for a C++ programmer._

Comment (eguiraud) [reply contribute/index.md:45]: yes that was a comment for you(s), not an edit suggestion :D 

To elaborate a bit: it's about managing expectations. we have seen many times that the "good first issue" label can mislead inexperienced developers in search of some "github street cred" into thinking that they can approach an issue, when it's not actually true, to their frustration as well as ours.

Just look at our list at https://github.com/root-project/root/labels/good%20first%20issue and think which of those could be picked up by the "mean square junior dev" without a lot of coaching. I think maybe 1 or 2.

Comment (couet) [reply contribute/index.md:45]: I totally agree. I will try to phrase it.

**Pull request merged**